lib/binance_api/base.rb

```diff
@@ -76,17 +75,17 @@ def map_params(options)
     end
 
     def snake_case_to_camel_case(key)
-      ['_', '\s'].each do |s|
-        key = key.to_s.gsub(/(?:#{s}+)([a-z])/){ $1.upcase }
+      %w[_ \s].each do |s|
```

Comment:
What do you do here? Can you describe each change and reason?

Reply:
Reason: rubocop. It's the same code, but in rubocop accepted format.